src/platform_impl/macos/event_loop.rs

```diff
@@ -23,6 +33,38 @@ use crate::{
     },
 };
 
+//pub type PanicData = Option<Box<dyn Any + Send + 'static>>;
+
+#[derive(Default)]
+pub(crate) struct PanicInfo {
+    inner: Cell<Option<Box<dyn Any + Send + 'static>>>,
+}
+// ------------------------------------
+// WARNING:
+// This struct is UnwindSafe if `get_mut` is never called on the inner data.
```

Comment:
If I'm reading this right, we're safe from this as long as no `get_mut` usages are in that `PanicInfo` impl block below, or is the scope larger than that?

Reply:
I think you are reading this right. Of course, the rest of the module can also directly access the private `inner` so that's why I choose the specific wording. Although I can see that it's a strange way to say what I'm trying to say here, so I just made an attempt at improving this.

Reply:
This reads very clearly now, thanks!